### ISSUE TITLE
Training script for BERT-based classifiers

### ISSUE THREAD

**Issue by asmithh** [davidatbu/openFraming#27]
Given a training set file (I'm going to try to support csv, tsv, and Excel) with a content column and a label column, we fine-tune and save a BERT-based classifier.
